### PR TITLE
chore(release):  bump version

### PR DESCRIPTION
**Pull request by StephanH90**

### CODE REVIEW

**Review by open-dynaMIX — CHANGES_REQUESTED** [review]
Why is this a major version bump? Should be `2.0.1`, right?

**Comment by StephanH90** [comment]
We are using the "newer" filter syntax. I mean its pretty far fetched since nobody is using such an old caluma version anyway (I hope?)








--

Adfinis AG
Stephan Hug, Software Engineer, GPG Key: C831A3A24D5719BA619604E622CD5AB4CFECD397
Giessereiweg 5 | CH-3007 Berne
Office +41 31 550 31 11 | Direct +41 31 550 31 02
www.adfinis.com










-----Original message-----

**Comment by open-dynaMIX** [comment]
Ah, I see. Then it would be good to add this info to the changelog.

**Comment by StephanH90** [comment]
I wanted to, but I was looking at the changelog of caluma and I couldn't find when it was changed. I went as far back as v5. So I wasn't sure if it was worth the effort to dig further.



Perhaps we can just write something like: "requires Caluma > v5" or similar.








--

Adfinis AG
Stephan Hug, Software Engineer, GPG Key: C831A3A24D5719BA619604E622CD5AB4CFECD397
Giessereiweg 5 | CH-3007 Berne
Office +41 31 550 31 11 | Direct +41 31 550 31 02
www.adfinis.com










-----Original message-----

**Comment by open-dynaMIX** [comment]
I just dug through the commit history of caluma. The new-style filter-syntax has been introduced with [this PR](https://github.com/projectcaluma/caluma/pull/708), merged on Oct 10, 2019. The first release with this changes must therefore be  [v3.0.0](https://github.com/projectcaluma/caluma/releases/tag/v3.0.0). This somehow didn't make it into the release notes, tho...

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+# v3.0.0 (2022-10-26)
+
+### Breaking
+
+* Due to the usage of the newer style filter syntax for queries (6e63070064e2f72ae3226a919bbecf876e31d26f) Caluma V5 or newer is required
```
Comment (open-dynaMIX) [inline CHANGELOG.md:5]:
```suggestion
* Due to the usage of the newer style filter syntax for queries (6e63070064e2f72ae3226a919bbecf876e31d26f) Caluma V3 or newer is required
```

**Review by open-dynaMIX — APPROVED** [review]
One last nitpick, sorry! :see_no_evil:

File: CHANGELOG.md (hunk 1 above)
Comment (StephanH90) [reply CHANGELOG.md]: Done. Pls check and resolve thx. :100:

**Pull request merged**